cico_run_tests.sh

```diff
@@ -15,16 +15,17 @@ docker run --detach=true -t \
     -v $(pwd):$PACKAGE_PATH:Z \
     -u $(id -u $USER):$(id -g $USER) \
     -e GOPATH=$GOPATH_IN_CONTAINER \
+    -e USE_GO_VERSION_FROM_WEBSITE=1 \
```

Comment:
Why do we need that?

Reply:
because I was too lazy to correctly check the real cause of the previous failure - I thought that the version was required by traefik :-)
fixed 2c92885